src/locale/locales/de/messages.po

```diff
@@ -403,46 +393,45 @@ msgstr "Anmeldung zur Warteliste abbrechen"
 msgid "Change"
 msgstr "Ändern"
 
-#: src/view/screens/Settings.tsx:601
-#: src/view/screens/Settings.tsx:610
+#: src/view/screens/Settings.tsx:601 src/view/screens/Settings.tsx:610
 msgid "Change handle"
-msgstr "Änderungsgriff andern"
+msgstr "Handle ändern"
 
 #: src/view/com/modals/ChangeHandle.tsx:161
 msgid "Change Handle"
-msgstr "Änderungsgriff andern"
+msgstr "Handle ändern"
 
 #: src/view/com/modals/VerifyEmail.tsx:141
 msgid "Change my email"
 msgstr "Meine E-Mail ändern"
 
 #: src/view/com/modals/ChangeEmail.tsx:109
 msgid "Change Your Email"
-msgstr "Ihre E-Mail ändern"
+msgstr "Deine E-Mail ändern"
 
 #: src/view/com/auth/onboarding/RecommendedFeeds.tsx:121
 msgid "Check out some recommended feeds. Tap + to add them to your list of pinned feeds."
-msgstr "Sehen Sie sich einige empfohlene Feeds an. Tippen Sie auf +, um sie zu Ihrer Liste der angehefteten Feeds hinzuzufügen."
+msgstr "Schau dir einige empfohlene Feeds an. Tippe auf +, um sie zu deiner Liste der angehefteten Feeds hinzuzufügen."
 
 #: src/view/com/auth/onboarding/RecommendedFollows.tsx:185
 msgid "Check out some recommended users. Follow them to see similar users."
-msgstr "Sehen Sie sich einige empfohlene Benutzer an. Folgen Sie ihnen, um ähnliche Benutzer zu sehen."
+msgstr "Schau dir einige empfohlene Nutzer an. Folge ihnen, um ähnliche Nutzer zu sehen."
 
 #: src/view/com/modals/DeleteAccount.tsx:163
 msgid "Check your inbox for an email with the confirmation code to enter below:"
-msgstr "Prüfen Sie Ihren Posteingang auf eine E-Mail mit dem Bestätigungscode, den Sie unten eingeben müssen:"
+msgstr "Überprüfe deinen Posteingang auf eine E-Mail mit dem Bestätigungscode, den du unten eingeben musst:"
 
 #: src/view/com/modals/ServerInput.tsx:38
 msgid "Choose Service"
-msgstr "Service auswählen"
+msgstr "Service wählen"
 
 #: src/view/com/auth/onboarding/WelcomeMobile.tsx:83
 msgid "Choose the algorithms that power your experience with custom feeds."
-msgstr "Wählen Sie die Algorithmen, die Ihr Erlebnis mit benutzerdefinierten Feeds unterstützen."
+msgstr ""
```

Comment:
This is in public marketing (on-boarding) context, so it might be important to refine this. An okayish Suggestion:
`"Wähle selbst Algorithmen aus, die dein Erlebnis mit personalisierten Feeds bereichern."`

Reply:
Technically correct, but a strange thing to say.

Reply:
I think it is meant as follows: "Wähle auf unterschiedlichen Algorithmen basierende, personalisierbare Feeds für ein neues Benutzererlebnis."
(Choose personalizable feeds based on different algorithms for a new user experience.)

Reply:
Or just: "Verbessere dein Nutzererlebnis mit eigenen Algorithmen und Feeds."

Reply:
> "Wähle Algorithmen, die für dich relevante Inhalte in deinem Feed anzeigen"
("Choose algorithms, that display posts relevant to you in your feed")